plugins/pipelines/templates/updateTrusted.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const metaSchema = schema.models.templateMeta.base;
+
+module.exports = () => ({
+    method: 'PUT',
+    path: '/pipeline/templates/{namespace}/{name}/trusted',
+    options: {
+        description: "Update a pipeline template's trusted property",
+        notes: 'Returns null if successful',
+        tags: ['api', 'pipeline', 'templates', 'trusted'],
+        auth: {
+            strategies: ['token'],
+            scope: ['admin', '!guest']
+        },
+
+        handler: async (request, h) => {
+            const { namespace, name } = request.params;
+            const { pipelineTemplateFactory } = request.server.app;
+            const { trusted } = request.payload;
```

Comment:
the trusted API should also take `version` in the payload, to mark specific versions and lower as trusted